### PR TITLE
week1-homework

### PR DESCRIPTION
**Pull request by seenachacko**

### CODE REVIEW

File: week1/task1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿string input = "world";
+string reversed = ReverseString(input); 
+Console.WriteLine($"Reversed input value: {reversed}");
+
+string ReverseString(string input)
+{
+    char[] myArr = input.ToCharArray();
```
Comment (calindurnea) [inline week1/task1/Program.cs]: It's a good approach to avoid naming variables with `my`

File: week1/task2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿string input = "Intellectualization";
+int vowelCount = GetVowelCount(input);
+Console.WriteLine($"Number of vowels: {vowelCount}");
+
+int GetVowelCount(string input)
+{
+    int count = 0;
+    foreach (char letter in input)
+    {
+     if(letter == 'a' ||  letter == 'e' || letter == 'i' || letter == 'o' || letter == 'u' || letter == 'A' || letter == 'E' || letter == 'I' || letter == 'O' || letter == 'U')
```
Comment (calindurnea) [inline week1/task2/Program.cs]: You can avoid having to check for both lowercase and uppercase by, for example, doing `input.ToLower()`

File: week1/task2/Program.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+int GetVowelCount(string input)
+{
+    int count = 0;
+    foreach (char letter in input)
+    {
+     if(letter == 'a' ||  letter == 'e' || letter == 'i' || letter == 'o' || letter == 'u' || letter == 'A' || letter == 'E' || letter == 'I' || letter == 'O' || letter == 'U') 
+     {
+        count ++;
+     }
```
Comment (calindurnea) [inline week1/task2/Program.cs]: Formatting here looks a bit strange, try using the shortcut `Shift + Alt + F` in VS code

File: week1/task3/Program.cs [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+Console.WriteLine($"Sum of negative numbers: {result[0]}. Multiplication of positive numbers: {result[1]}");
+
+int[] GetResult(int[] input)
+{
+    int multiplicationResult = 1;
+    int sumofNegativeNumbers = 0;
+
+    foreach (int number in input)
+    {
+        if (number >= 0)
```
Comment (calindurnea) [inline week1/task3/Program.cs]: here if the number is 0 then the multiplication result will also be 0, the check should be strict > than 0

**Review by calindurnea — COMMENTED** [review]
Very good! 💯 
Left you a few comments, keep up the good work! 👍

**Comment by Vertygo** [comment]
@seenachacko do you need some help in regards to review comments?

File: week1/task3/Program.cs (hunk 4 above)
Comment (seenachacko) [reply week1/task3/Program.cs]: Thank you for your feedback.

**Comment by seenachacko** [comment]
> @seenachacko do you need some help in regards to review comments?
 Thanks , I have updated the code based on feedback.

**Review by calindurnea — APPROVED** [review]

**Pull request merged**